stdlib/2.7/os/__init__.pyi

```diff
@@ -134,7 +134,16 @@ def rmdir(path: unicode) -> None: ...
 # TODO(MichalPokorny)
 def stat(path: unicode) -> Any: ...
 
-# TODO: stat_float_times, statvfs, tempnam, tmpnam, TMP_MAX, walk
+# TODO: stat_float_times, statvfs, tempnam, tmpnam, TMP_MAX
+def walk(top: AnyStr, topdown: bool = ..., onerror: Any = ...,
+         followlinks: bool = ...) -> Iterator[Tuple[AnyStr, List[AnyStr],
+                                                      List[AnyStr]]]: ...
```

Comment:
This indent seems off. Also, we don't usually include docs/comments in stubs unless there's something funky with the stub. If it's about the `onerror: Any`, maybe explain why it can't be `Callable[[OSError], None]`?